### ISSUE TITLE
Question: How can I customize coin symbol

### ISSUE THREAD

**Issue by rjl493456442** [poanetwork/blockscout#1387]
As the title described, how can I customize the coin symbol? 

I use the `COIN` environment variable to set the coin as `ETH`, but it seems only to affect the market value, but doesn't change the coin symbol.

![image](https://user-images.githubusercontent.com/5959481/51966326-2c20db80-24a7-11e9-8765-ae1b84339b03.png)

The coin symbol in the screenshot should be `ETH`

**Comment by acravenho** [comment]
@rjl493456442 

Most of the text in BlockScout utilizes `gettext` which creates `po` and `pot` files in the `BlockScoutWeb` Umbrella application.

To edit these values, check the msgid/msgstr in the following files:

https://github.com/poanetwork/blockscout/blob/master/apps/block_scout_web/priv/gettext/en/LC_MESSAGES/default.po

https://github.com/poanetwork/blockscout/blob/master/apps/block_scout_web/priv/gettext/default.pot

The default text will be listed as

 ```
msgid "ETH"
msgstr ""
```


For your particular scenario, remove the POA listed on this line:

https://github.com/poanetwork/blockscout/blob/03061be2f710a1fc84c6ce555f676e8bb9dfa54d/apps/block_scout_web/priv/gettext/en/LC_MESSAGES/default.po#L405

**Comment by igorbarinov** [comment]
Let's add it to FAQ section on the forum

**Comment by rjl493456442** [comment]
After I change `msgstr "POA"` to `msgstr ""` in line 405, it works!

```
--- a/apps/block_scout_web/priv/gettext/en/LC_MESSAGES/default.po
+++ b/apps/block_scout_web/priv/gettext/en/LC_MESSAGES/default.po
@@ -402,7 +402,7 @@ msgstr ""
 #: lib/block_scout_web/templates/transaction/overview.html.eex:104
 #: lib/block_scout_web/views/wei_helpers.ex:72
 msgid "Ether"
-msgstr "POA"
+msgstr ""
```

But when the container starts, it will print such error message and the Transaction frame is empty.

```
INFO [01-31|02:38:58.018] Imported new chain segment               blocks=1    txs=0 mgas=0.000 elapsed=1.026ms   mgasps=0.000 number=39348 hash=6eaf17…e47dd6 cache=1.88kB
2019-01-31T02:38:58.053 fetcher=block_realtime [error] GenServer Indexer.Block.Realtime.Fetcher terminating
** (ArithmeticError) bad argument in arithmetic expression
    (indexer) lib/indexer/block/realtime/fetcher.ex:218: Indexer.Block.Realtime.Fetcher.determine_start_at/3
    (indexer) lib/indexer/block/realtime/fetcher.ex:202: Indexer.Block.Realtime.Fetcher.start_fetch_and_import/4
    (indexer) lib/indexer/block/realtime/fetcher.ex:87: Indexer.Block.Realtime.Fetcher.handle_info/2
    (stdlib) gen_server.erl:637: :gen_server.try_dispatch/4
    (stdlib) gen_server.erl:711: :gen_server.handle_msg/6
    (stdlib) proc_lib.erl:249: :proc_lib.init_p_do_apply/3
Last message: {%EthereumJSONRPC.Subscription{reference: #Reference<0.2327056874.1126957057.186732>, subscriber_pid: #PID<0.2321.0>, transport: EthereumJSONRPC.WebSocket, transport_options: %EthereumJSONRPC.WebSocket{url: "ws://localhost:8546", web_socket: EthereumJSONRPC.WebSocket.WebSocketClient, web_socket_options: %EthereumJSONRPC.WebSocket.WebSocketClient.Options{event: "newHeads", params: [], web_socket: #PID<0.676.0>}}}, {:ok, %{"difficulty" => "0x2", "extraData" => "0xd883010814846765746888676f312e31312e32856c696e757800000000000000272c5349cd62a210be3236c86409584d27b97df6c4eb7e0a26fa8183fa2201a337bd3a2c4c8a60ad77f83660f999f0e13178ccd8049b3af7ac188d31b2f07f8701", "gasLimit" => "0x7270e0", "gasUsed" => "0x0", "hash" => "0x6eaf175b4b04da2e14564a132c5e2d24a6d38eb4f604d77edceba19832e47dd6", "logsBloom" => "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000", "miner" => "0x0000000000000000000000000000000000000000", "mixHash" => "0x0000000000000000000000000000000000000000000000000000000000000000", "nonce" => "0x0000000000000000", "number" => "0x99b4", "parentHash" => "0xbb6edc18bdd91895d8b077310a9e85d6040b2e6df13ab4002a753ffa476fcbbc", "receiptsRoot" => "0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421", "sha3Uncles" => "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347", "stateRoot" => "0x137a2eb66ec5e7aa741017f4e4da52261284d2d72097b3014186564e6e57d269", "timestamp" => "0x5c525fc2", "transactionsRoot" => "0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421"}}}
State: %Indexer.Block.Realtime.Fetcher{block_fetcher: %Indexer.Block.Fetcher{broadcast: :realtime, callback_module: Indexer.Block.Realtime.Fetcher, json_rpc_named_arguments: [transport: EthereumJSONRPC.HTTP, transport_options: [http: EthereumJSONRPC.HTTP.HTTPoison, url: "http://localhost:8545", http_options: [recv_timeout: 60000, timeout: 60000, hackney: [pool: :ethereum_jsonrpc]]], variant: EthereumJSONRPC.Geth], receipts_batch_size: 250, receipts_concurrency: 10}, max_number_seen: 39347, previous_number: nil, subscription: %EthereumJSONRPC.Subscription{reference: #Reference<0.2327056874.1126957057.186732>, subscriber_pid: #PID<0.2321.0>, transport: EthereumJSONRPC.WebSocket, transport_options: %EthereumJSONRPC.WebSocket{url: "ws://localhost:8546", web_socket: EthereumJSONRPC.WebSocket.WebSocketClient, web_socket_options: %EthereumJSONRPC.WebSocket.WebSocketClient.Options{event: "newHeads", params: [], web_socket: #PID<0.676.0>}}}, timer: #Reference<0.2327056874.1126957058.227444>}
```

And **about one minute later**, this type of error message disappears and the Transaction frame begins to show the transactions information.

I didn't see this error message before, is it relative with my changes?

**Comment by acravenho** [comment]
@rjl493456442 https://github.com/poanetwork/blockscout/blob/32107089d5b03ace9ec2f89dc9fe3f6386422bef/apps/indexer/lib/indexer/block/realtime/fetcher.ex#L212-L231

This is trying to determine where to start the realtime fetcher. I haven't seen this error but will keep an eye out for it. The change you made wouldn't affect this error.

**Comment by acravenho** [comment]
Resolved by #1409
